packages/sanity/src/structure/panes/document/documentPanel/header/perspective/DocumentPerspectiveMenu.tsx

```diff
@@ -7,24 +7,29 @@ import {
   BundleMenu,
   DEFAULT_STUDIO_CLIENT_OPTIONS,
   getAllVersionsOfDocument,
-  LATEST,
   useClient,
   usePerspective,
 } from 'sanity'
+import {useRouter} from 'sanity/router'
+import {styled} from 'styled-components'
 
-// FIXME
-// eslint-disable-next-line boundaries/element-types
+import {getVersionName} from '../../../../../../core/bundles'
 import {useBundles} from '../../../../../../core/store/bundles/useBundles'
 
+const BadgeButton = styled(Button)({
```

Comment:
Just wondering; do we have a good reason for overriding the behaviour of the UI library here? I have no strong opinion about the right choice, but I do think things like this should probably be consistent across the UI.

Reply:
tbh I did it because that's how Marius had it in his prototype and while doing it (before adding this) without this cursor affordance it's really hard to tell that this is a clickable element. 

I'm of the opinion that we should likely have this everywhere on the studio but last time the cursor discourse was brought up nothing was really done about it 😄

Reply:
> without this cursor affordance it's really hard to tell that this is a clickable element.

I'm always on the fence about it, but usually land on the side of respecting the way a user's entire operating system handes buttons. But it's an age old debate and totally not worth bringing up again 🤣.

Let's discuss it no further!

Reply:
I'll add it to my notes to bring up to Marius :) (I have a linear story for it)